polylogue/importers/codex.py

```
@@ -71,23 +56,79 @@ def import_codex_session(
-        for raw in handle:
-            entry = json.loads(raw)
+        for line_idx, raw in enumerate(handle, start=1):
             try:
-                model = _ResponseItem.model_validate(entry)
-            except ValidationError:
+                entry = json.loads(raw)
+            except json.JSONDecodeError:
+                fallback = raw.strip()
+                if not fallback:
+                    continue
+                chunk_text = "Unparsed Codex log line\n```\n{}```".format(fallback)
```

Ruff (0.15.9)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)